openmc/plotter.py

```diff
@@ -27,10 +27,37 @@
 UNITY_MT = -1
 XI_MT = -2
 
+def get_reaction_components(mt):
+    """Determine what reactions make up redundant reaction.
+
+    Parameters
+    ----------
+    mt : int
+        ENDF MT number of the reaction to find components of.
+
+    Returns
+    -------
+    mts : list of int
+        ENDF MT numbers of reactions that make up the redundant reaction and
+        have cross sections provided.
+
+    """
+    mts = []
+    if mt in openmc.data.SUM_RULES:
+        for mt_i in openmc.data.SUM_RULES[mt]:
+            mts += get_reaction_components(mt_i)
```

Comment:
This could be reduced using [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get), e.g.
```python
for mt_i in openmc.data.SUM_RULES.get(mt, []):
```